src/index.js

```diff
@@ -51,13 +53,14 @@ const ping = (options, shouldMute) => {
     // Fires all requests
     .then(response => {
       log(`Loading details of the latest ${options.limit} top stories`);
-
-      return fetchTopStoriesDetails(response, options.limit);
+      return fetchTopStoriesDetails(response);
     })
     // Returns a formatted array with the response request
     .then(response => {
       return response.map(item => item.body);
     })
+    // Sort the response if options.latest requested
+    .then(response => options.latest ? response.slice().sort(sortByTime) : response)
```

Comment:
@erickbelfy Sorry about my OCD but can you wrap these into brackets? Just so it looks consistent. Other than that this PR is looking perfect :muscle: